### ISSUE TITLE
After applying the reflow, linkedin, youtube and github links are not visible: A11y_WCP URLs - ONNX Runtime_ONNX Runtime- Home_Top navigation_Reflow

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20119]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11; #FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; #Reflow; #WCAG1.4.10; #A11yMAS;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".** 

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2

### Pre-Requisites:

1. Open the web page / screen of the web application that is to be tested.
2. Open Chrome/Edge dev Inspect tools. Keyboard shortcut "Ctrl+Shift+I".
3. Ensure devtools are customized to Dock location: "undock into separate window". Minimize devtools so they are not obscuring the page.
4. Zoom the browser window of the page to be tested to 200%.
5. Adjust the browser window's height and width to the required viewport of 320px*256px. As long as dev tools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified.
6. Refresh the page.

**Alternatively:**

1. Change the OS display settings to a resolution of 1280x1024 at 100% dpi scaling.
2. Open the web page/ screen of the web application that is to be tested.
3. Zoom browser 400%.
4. Set the browser to full screen. Keyboard shortcut "F11" key.
5. If any browser sidebars are still visible (e.g., the Edge discover sidebar), these must be closed.
### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page is appeared.
3. Tab till "Top navigation" section.
4. Tab till "Community" drop down.
5. Now check that after applying the reflow, LinkedIn, YouTube and GitHub links are not visible.

### Actual Results:
After applying the reflow, LinkedIn, YouTube and GitHub links are not visible.

### Expected Results:
After applying the reflow, LinkedIn, YouTube and GitHub links should be visible without any overlap and truncations.

### User Impact:
Low visible user will face difficultly. There not able to see the LinkedIn, YouTube and GitHub links. If links are not visible in reflow.

### Attachment:

[python, linkedin, youtube and github links are not visible.zip](https://github.com/microsoft/onnxruntime/files/14788942/python.linkedin.youtube.and.github.links.are.not.visible.zip)

![top navigation](https://github.com/microsoft/onnxruntime/assets/142479541/d9718707-c000-4183-b969-81315a96893c)

**Comment by MaanavD** [comment]
note: I made it so the top navbar is not sticky at really small window sizes, enabling users to see the entire content by scrolling.

**Comment by PadminiGariki** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.97 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
#Regressed:04-18-24; #Closed;

For reference attachment is added.

![Screenshot 2024-04-18 115444](https://github.com/microsoft/onnxruntime/assets/142479541/bd5226fe-4e09-46ca-9f65-1486399e094a)
